test/utils.py

```diff
@@ -127,15 +127,21 @@ def verify_testrunner(testrunner, ts0, ts1='', ts2='', ts3='', tr=''):
 
 def expect_testrunner_pass(logfile_path):
     with open(logfile_path) as file, mmap.mmap(file.fileno(), 0, access=mmap.ACCESS_READ) as log:
-        return re.search(br'INFO:  \[.*\]\[testrunner\]: PASSED (. of . suites passing) \[$SVUnitVersion\]', log)
+        if re.search(br'INFO:  \[.*\]\[testrunner\]: PASSED \(. of . suites passing\) \[$SVUnitVersion\]', log):
```

Comment:
Wouldn't `assert re.search(...)` also do the trick?

Reply:
Hi
As always in Python, everything can be done in various ways :)
but yes with `assert` code looks nicer
Even more, what do you think about such implementation:
`assert re.search(pattern, log), "\"%s\" not found at %s log file" % (pattern, logfile_path)`
This will give us clear evaluated pattern string in error message

Reply:
Already done

Reply:
@tudortimi please resolve if it's good for you